src/components/CourseCard/CourseCard.jsx

```diff
@@ -0,0 +1,119 @@
+import {
+  Button,
+  Card,
+  CardActions,
+  CardContent,
+  CardMedia,
+  Collapse,
+  List,
+  ListItem,
+  ListItemText,
+  Typography,
+} from '@mui/material';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import React, { useState } from 'react';
+import ReactPlayer from 'react-player';
+import { NavLink } from 'react-router-dom';
+import { save } from '../../services/localStorage/storage.js';
+import { ExpandMore } from './CourseCard.styled';
+
+const CourseCard = ({ data }) => {
+  const [hovered, setHovered] = useState(false);
+  const [expanded, setExpanded] = useState(false);
+
+  const handleExpandClick = () => {
+    setExpanded(!expanded);
+  };
+
+  const { id, title, previewImageLink, lessonsCount, rating, meta } = data;
```

Comment:
```suggestion
  const { 
    id, 
    title, 
    previewImageLink, 
    lessonsCount, 
    rating, 
    meta,
  } = data;
```
Мені таке форматування здається зручнішим, адже пропсів досить багато

Reply:
за таким должен следить линтер.  которого нет)

Reply:
> 
Не знаю, в мене prettier форматить довгий запис пропсів - в даному випадку, список не настільки великий, за яким би воно вже вирівнювалося на кожній строчці)